packages/core/src/Stores/client-store.js

```diff
@@ -387,6 +389,7 @@ export default class ClientStore extends BaseStore {
             fetchFinancialAssessment: action.bound,
             setFinancialAndTradingAssessment: action.bound,
             setTwoFAStatus: action.bound,
+            setTwoFAChangedStatus: action.bound,
```

Comment:
@shaheer-deriv Please update the types in `packages/stores/types.ts` 🙏🏻

Reply:
@farzin-deriv added 😄 👍